plugin/trino-faker/src/test/java/io/trino/plugin/faker/FakerQueryRunner.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.faker;
+
+import io.airlift.log.Level;
+import io.airlift.log.Logger;
+import io.airlift.log.Logging;
+import io.trino.Session;
+import io.trino.testing.DistributedQueryRunner;
+import io.trino.testing.QueryRunner;
+
+import java.util.Map;
+
+import static io.trino.testing.TestingSession.testSessionBuilder;
+import static java.util.Objects.requireNonNullElse;
+
+public class FakerQueryRunner
+{
+    private FakerQueryRunner() {}
+
+    public static QueryRunner createQueryRunner()
+            throws Exception
+    {
+        Session defaultSession = testSessionBuilder()
+                .setCatalog("faker")
+                .setSchema("default")
+                .build();
+
+        QueryRunner queryRunner = DistributedQueryRunner.builder(defaultSession)
+                .setExtraProperties(Map.of(
+                        "http-server.http.port", requireNonNullElse(System.getenv("TRINO_PORT"), "8080")))
+                .setWorkerCount(0)
```

Comment:
no need to force 0 workers

Reply:
This is needed when setting the `http-server.http.port` value for all nodes, and currently the connector doesn't use more than one node anyway.